tutorials/s4sdk-secure-cloudfoundry/s4sdk-secure-cloudfoundry.md

```diff
@@ -194,6 +194,10 @@ After logging in you should see the `HelloWorld` servlet which is now served by
 [ACCORDION-BEGIN [Step 3: ](Protect your backend microservice)]
 After authentication works with the App Router, your java backend service is still fully visible in the web and not protected. Therefore, we need to protect our java microservices as well so that they accept requests with valid `JWTs` for the current user only. In addition, we will setup the microservice in a way that it deals with authorization, i.e., understands the OAuth scopes from the JWT that we have configured previously using the `xs-security.json` file.
 
+[OPTION BEGIN [Apache]]
```

Comment:
@thecodester Unfortunately I cannot use `TomEE` here because of the spellchecker. Could you add it to the dictionary or can I somehow else work around this?

Reply:
@MichaelCzcz do you have an idea how to improve this?